lib/gtfs_server/agencies/trimet/client.ex

```diff
@@ -0,0 +1,49 @@
+defmodule GtfsServer.Agencies.Trimet.Client do
+  use HTTPoison.Base
+
+  @endpoint "http://developer.trimet.org/ws/V1"
+  @trimet_app_id Application.get_env(:gtfs_server, :trimet_app_id)
+
+  alias GtfsServer.Protobufs.FeedMessage
+  alias GtfsServer.Agencies.Trimet.Client
+
+  require Logger
+
+  @moduledoc """
+  GtfsProducer.TrimetRT service is responsible for wrapping the Trimet GTFS-RT
+  API so that Trimet GTFS-RT data can be injested to the platform.
+  """
+  def get_feed(:trimet_vehicle_posistion_feed) do
+    with {:error, message} <- fetch("/VehiclePositions")  do
+      Logger.error("Trimet GTFS RT feed: #{message}")
```

Comment:
Returning :ok Iirc from the logger, this might cause some weirdness down the chain and also removes the ability for the caller to handle the errors